tests/integration/pilot/multi_version_revision_test.go

```diff
@@ -93,13 +97,18 @@ func TestMultiVersionRevision(t *testing.T) {
 						{
 							Name:         "http",
 							Protocol:     protocol.HTTP,
-							InstancePort: 8080,
+							InstancePort: 8000,
```

Comment:
Not sure why this port was changed, but otherwise LGTM.

Reply:
Ah accidentally left it in, shouldn't matter though since it's an arbitrary port